higher-order-scaffolders.md

```diff
@@ -0,0 +1,62 @@
+# Higher Order Scaffolders
+
+Start Date: 2020-08-16
+
+Author: @travi
+
+## Summary
+
+Define a pattern for scaffolders that can follow a more rigid process than
+simple plugins, like package or application scaffolders. Because they solve a
+well defined problem, they are able to define answers to the generic questions
+provided by default in the [project-scaffolder](https://github.com/travi/project-scaffolder)
+or language scaffolders. These also need to define details commonly returned as
+results from the plugin/sub-scaffolders, like `dependencies`,
+`devDependencies`, `scripts`, and others.
+
+## Motivation
+
+* Further automate well-defined scenarios that have become repetitive
```

Comment:
👍 think it makes a lot of sense to make something like this easy. feel like most consumers (whether a company or specific user) would have very consistent answers they always want to use.

Reply:
looks like i never got this detail documented, so i want to make sure i shared with you in the past that it is possible to pass `decisions` [to the project-scaffolder](https://github.com/travi/project-scaffolder/blob/bcaaf7c350db1f9a821ef385dd59da93853409fb/src/scaffolder.js#L19) and [to the js-scaffolder](https://github.com/travi/javascript-scaffolder/blob/05c19172e7674872c49dfa706d4d3ab4cd648129/src/scaffolder.js#L36) and others. i enabled this partly to improve determinism of integration tests, but also to enable skipping and error proofing prompts that are answered consistently for an org. here is an example [for form8ion's cli](https://github.com/form8ion/utils-cli/blob/ddc5aad78b2d684255bc515aa8e6b9f1d373c110/src/commands/scaffold/command.js#L10-L20).

just wanted to make sure you were aware of that piece already, so that it is clear what the distinction is for the desired new behavior in this rfc